sessiontxn/txn_context_test.go

```diff
@@ -729,102 +729,66 @@ func TestStillWriteConflictAfterRetry(t *testing.T) {
 		"update t1 set v=v+1 where id in (1, 2, 3) and v>0",
 	}
 
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
 	for _, isolation := range []string{ast.RepeatableRead, ast.ReadCommitted} {
 		for _, query := range queries {
 			for _, autocommit := range []bool{true, false} {
 				t.Run(fmt.Sprintf("%s,%s,autocommit=%v", isolation, query, autocommit), func(t *testing.T) {
-					testStillWriteConflictAfterRetry(t, store, isolation, query, autocommit)
+					tk.MustExec("truncate table t1")
+					tk.MustExec("insert into t1 values(1, 10)")
+					tk2 := testkit.NewSteppedTasksRunner(t).CreateSteppedTestKit("s2", store)
```

Comment:
Why can not it be ` testkit.NewSteppedTestKit(t, "s2", store)`? Could we register a global runner? Or can we make it possible to parallel running multiple stepped task?